eatgo-admin-api/src/test/java/kr/co/fastcampus/eatgo/application/RestaurantServiceTests.java

```diff
@@ -64,9 +65,11 @@ public void getRestaurantWithExisted() {
         assertThat(restaurant.getId()).isEqualTo(1004L);
     }
 
-    @Test(expected = RestaurantNotFoundException.class)
+    @Test
     public void getRestaurantWithNotExisted() {
-        restaurantService.getRestaurant(404L);
+        assertThatThrownBy(() -> {
+            restaurantService.getRestaurant(404L);
+        }).isInstanceOf(RestaurantNotFoundException.class);
     }
```

Comment:
Use `assertThatThrownBy` instead of `@Test(expected = Exception.class`.

See [AssertJ Exception assertions guide](https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion).